template/deploy.sh

```diff
@@ -0,0 +1,31 @@
+# You will need the heroku-cli:
+# https://devcenter.heroku.com/articles/heroku-cli
+
+HEROKU_FRONTEND_NAME=
+HEROKU_BACKEND_NAME=
```

Comment:
I suppose it would be better to set these in an `.env...` file, then the "setup" and "push" could be separate scripts.  Maybe mark that as a  TODO.